kinder/roadmap.md

```diff
@@ -0,0 +1,45 @@
+# Roadmap 🗺️
+
+This document outlines some goals, non-goals, and future aspirations for kinder.
+
+kinder is an example of [kind](https://github.com/kubernetes-sigs/kind) used as a library.
+
+All the kind commands will be available in kinder, side by side with additional commands designed
+for helping kubeadm contributors.
+
+High level goals for kinder v0.1 include:
+
+- [ ] Provide a local test environment for kubeadm development
+   - [x] Allow creation of nodes "ready for installing Kubernetes"
```

Comment:
this one / phases will also hopefully land upstream O(soon), been prototyping various options for this in kind :sweat_smile:

Reply:
Great! I'm happy to nuke this in kinder as soon as possible!